diff -urN aaa/Clock/rox_run Clock/rox_run
--- aaa/Clock/rox_run	Mon Mar  4 12:21:35 2002
+++ Clock/rox_run	Mon Jul  1 22:52:41 2002
@@ -15,11 +15,11 @@
 fi
 
 if [ -z "$LIBDIRPATH" ]; then
-    LIBDIRPATH=$HOME/lib:/usr/local/lib:/usr/lib export LIBDIRPATH
+    LIBDIRPATH=$HOME/lib:/usr/local/lib:/usr/lib:/usr/X11R6/lib export LIBDIRPATH
 fi
 
 if [ -z "$APPPATH" ]; then
-    APPPATH=$HOME/Apps:/usr/local/apps export APPPATH
+    APPPATH=$HOME/Apps:/usr/local/apps:/usr/X11R6/share/ROX-apps export APPPATH
 fi
 
 IFS=:
